qa/tasks/cephfs/test_multimds_misc.py

```diff
@@ -131,17 +137,15 @@ def test_scrub_abort_mds0(self):
             file_obj_name = "{0:x}.00000000".format(ino)
             self.fs.rados(["rmxattr", file_obj_name, "parent"])
 
-        out_json = self.fs.rank_tell(["scrub", "start", "/d1/d2/d3", "recursive", "force"], 0)
+        out_json = self.fs.run_scrub(["start", "/d1/d2/d3", "recursive", "force"], 0)
         self.assertNotEqual(out_json, None)
         
-        res = self.fs.rank_tell(["scrub", "abort"])
+        res = self.fs.run_scrub(["abort"])
         self.assertEqual(res['return_code'], 0)
 
         # Abort and verify in both mdss. We also check the status in rank 0 mds because
         # it is supposed to gather the scrub status from other mdss.
-        self.wait_until_true(lambda: "no active" in self._get_scrub_status(1)['status']
-                and "no active" in self._get_scrub_status(2)['status']
-                and "no active" in self._get_scrub_status(0)['status'], 30)
+        self._check_scrub_status(result="no active")
```

Comment:
I believe these `no active` are `no active scrubs running` as per `ScrubStack::scrub_status`. So why don't put the complete string (ambiguous)?

Reply:
Fixed it by using the default value.